aether-producer/producer/__init__.py

```diff
@@ -346,10 +394,12 @@ def handle_topic_command(self, request, status):
 
 
 class TopicStatus(enum.Enum):
+    INITIALIZING = 0  # Started by not yet operational
```

Comment:
by or but? :thinking:

Reply:
I've (finally) turned on the spell checker (for comments) in my text editor.